src/patient-lab-dashboard/table/report-table/report-table.component.tsx

```diff
@@ -75,6 +97,25 @@ const ReportTable = (props: ReportTableProps) => {
     return selfPatient
   }
 
+  useEffect(() => {
+    if (testResults) {
+      setLabTestResults({...testResults})
+      const labOrder = testResults?.data?.results[0]
+      labOrder &&
+        getTestsInLabOrder(labOrder)?.map(sample => {
```

Comment:
I don't think map is being used correctly here, if the purpose is to loop through, please use forEach or a for loop

Reply:
introduced foreach instead of map